gensim/models/doc2vec.py

```diff
@@ -600,18 +600,20 @@ def __init__(self, documents=None, size=300, alpha=0.025, window=8, min_count=5,
           of the model.
 
         """
+
         super(Doc2Vec, self).__init__(
-            size=size, alpha=alpha, window=window, min_count=min_count, max_vocab_size=max_vocab_size,
-            sample=sample, seed=seed, workers=workers, min_alpha=min_alpha,
-            sg=(1+dm) % 2, hs=hs, negative=negative, cbow_mean=dm_mean,
+            size=size, window=window, min_count=min_count,
+            sample=sample, workers=workers,
```

Comment:
The use of `**kwargs` handles this. If you pass `workers=5` to Doc2Vec, if it's not explicitly declared in the `__init__()` signature, it stays in the `kwargs` dict. That's then passed to the superclass `__init__()` – overriding the default of 3. So the desired effect is achieved: shared defaults exist in one place (Word2Vec), but calls to either can override with an explicit valiue. 

(As noted in a prior comment, only `dm_mean` is especially tricky, needing a different approach, because it's a de facto synonym for `cbow_mean`. The goal is to have it so that if `dm_mean` is unspecified, the default Word2Vec `cbow_mean` value is operative, but if specified, it sets `cbow_mean` to the specified value. It's definitely also possible, and a solution might rely on the fact `kwargs` is a normal dict.)